portfolio/src/main/webapp/General.css

```diff
@@ -0,0 +1,64 @@
+/**
```

Comment:
nit: this multi-line comment can be on a single line

Reply:
fixed